mods/ts/rules/nod-structures.yaml

```diff
@@ -261,7 +261,7 @@ NASTLH:
 		Cost: 2500
 	Tooltip:
 		Name: Stealth Generator
-		Description: Generates a cloaking field
+		Description: Generates a cloaking field.
```

Comment:
+ "in a wide area around it to hide your forces from the enemy"
or something of the sort.